cmd/rotate_certs.go

```diff
@@ -451,6 +472,15 @@ func (rcc *rotateCertsCmd) rotateKubelet() error {
 			}
 		}
 	}
+
+	for _, host := range rcc.masterNodes {
+		log.Debugf("Restarting kubelet on node %s", host.Name)
+		out, err := rcc.sshCommandExecuter("sudo systemctl restart kubelet", rcc.masterFQDN, host.Name, "22", rcc.sshConfig)
```

Comment:
It would be nice if this had retry logic

Reply:
Would be nice to have but we will first need a way to check if it happened or not using ssh

Reply:
it can be done using ssh
create a follow up issue please to address this eventually